src/script/api/script_goal.hpp

```diff
@@ -170,7 +170,7 @@ class ScriptGoal : public ScriptObject {
 	 * @pre question != null && len(question) != 0.
 	 * @pre company == COMPANY_INVALID || ResolveCompanyID(company) != COMPANY_INVALID.
 	 * @pre CountBits(buttons) >= 1 && CountBits(buttons) <= 3.
-	 * @note Replies to the question are given by you via the event ScriptEventGoalQuestionAnswer.
+	 * @note Replies to the question are given by you via the event ScriptEvent_GoalQuestionAnswer.
```

Comment:
There is no class/event called `ScriptEvent_GoalQuestionAnswer`, though there is one called `ScriptEventGoalQuestionAnswer`.

Reply:
oh, how did that get in here...